File: BackEnd/FavouriteService/src/main/java/com/capstone/entity/FavouriteCuisine.java
package com.capstone.entity;

import lombok.*;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class FavouriteCuisine {
    @Id
    private int cuisineId;
    private String emailId;
    private String cuisineName;
    private String cuisineImage;
    private double price;
}

File: FrontEnd/src/app/model/favourite-cuisine.ts
export class FavouriteCuisine {
    public cuisineId : number;
    public cuisineName : string;
    public cuisineImage : string;
    public emailId : string;
    public price: number;
    
    constructor(cuisineId : number , cuisineName : string ,cuisineImage : string , emailId : string, price : number,) {
        this.cuisineId = cuisineId;
        this.cuisineName = cuisineName;
        this.cuisineImage = cuisineImage;
        this.emailId = emailId;
        this.price=price
    }
}
